main.py

```diff
@@ -0,0 +1,59 @@
+import sys
+
+import streamlit as st
+import torch
+from diffusers import StableDiffusionPipeline
+import requests
+from bs4 import BeautifulSoup
+from googletrans import Translator
+def check_url(url):
+    r = requests.head(url)
+    return r.status_code == 200
+
+def get_text_from_site(url):
+    try:
+        response = requests.get(url)
+        bs = BeautifulSoup(response.text, 'html.parser')
+        part_of_text = bs.find('div', 'c-detail text').text
+        index = part_of_text.lower().find("приметы")
+        i = part_of_text.find(":", index)
+        result_sentence = part_of_text[i+1:part_of_text.find("\n", i+1)].replace(".", "")
+        translator = Translator()
+        translation = translator.translate(result_sentence, src='ru', dest='en')
+        result_sentence = translation.text
+    except:
+        result_sentence = ""
```

Comment:
This is not user-friendly at all. Let's divide the function into parts. The first one will try to find information about a suspect. If there is no such information, just show a user a message and ask for another link. And if there is required information, you will pass it to another function and generate an image.